### DISCUSSION TITLE
How to set the Sliders rod arbitrary step length.

### DISCUSSION THREAD

**Discussion by Terracotta-62** [artisan-roaster-scope/artisan#1115] [Q&A]
![20230228061533](https://user-images.githubusercontent.com/116416332/221698322-ebd7cbcf-2556-43c2-817f-8f8bc9093bbe.png)

**Comment by Terracotta-62** [comment]
How do I set step 5

**Reply by MAKOMO** [reply]
Not possible. Step size of sliders is 1 (not coarse) or 10 (coarse)
